src/gui/torrentcontentmodel.cpp

```diff
@@ -88,6 +99,43 @@ namespace
             return QIcon(iconPixmap);
         }
     };
+#elif defined(Q_OS_MAC)
+    // There is a similar bug on macOS, to be reported to Qt
+    // https://github.com/qbittorrent/qBittorrent/pull/6156#issuecomment-316302615
+    class MacFileIconProvider: public UnifiedFileIconProvider
+    {
+    public:
+        using QFileIconProvider::icon;
```

Comment:
> Good, done. Hopefully somebody of the team gains love in clang-format one day.

The last time I checked (in 2016), clang-format was not able to reproduce guidelines of this project, so I did not bother to create a config for it. Maybe now it is capable to do this job.